source/agent/conference/roomController.js

```diff
@@ -617,15 +618,15 @@ module.exports.create = function (spec, on_init_ok, on_init_failed) {
         return null;
     };
 
-    var mixAudio = function (stream_id, view, on_ok, on_error) {
+    var mixAudio = function (stream_id, view, origin, on_ok, on_error) {
         log.debug('to mix audio of stream:', stream_id, 'mixed view:', view);
         var audio_mixer = getSubMediaMixer(view, 'audio');
         if (streams[stream_id] && audio_mixer && terminals[audio_mixer]) {
             var target_node = terminals[audio_mixer].locality.node,
                 spread_id = stream_id + '@' + target_node;
             spreadStream(stream_id, target_node, 'amixer', function() {
                 if (terminals[audio_mixer] && streams[stream_id]) {
-                    terminals[audio_mixer].subscribed[spread_id] = {audio: stream_id};
+                    terminals[audio_mixer].subscribed[spread_id] = {audio: stream_id, origin: origin};
```

Comment:
Why "origin" is saved in `terminals[audio_mixer].subscribed` not in `terminals[audio_mixer]`?